crates/forge/tests/fixtures/zk/Create2.t.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.18;
+
+import "forge-std/Test.sol";
+import {L2ContractHelper} from "era-contracts/l2-contracts/contracts/L2ContractHelper.sol"; // =0.8.20
+
+import {Greeter} from "../src/Greeter.sol";
+import {CustomNumber} from "../src/CustomNumber.sol";
+
+import {Create2Utils} from "../src/Create2Utils.sol";
+
+contract Create2Test is Test {
+    function getBytecodeHash(string memory path) internal returns (bytes32 bytecodeHash) {
+        string memory artifact = vm.readFile(path);
+        bytecodeHash = vm.parseJsonBytes32(artifact, ".hash");
+    }
+
+    function testCanDeployViaCreate2() public {
+        bytes32 bytecodeHash = getBytecodeHash("zkout/Greeter.sol/Greeter.json");
+        address sender = address(0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496);
+        bytes32 salt = "12345";
+        bytes32 constructorInputHash = keccak256(abi.encode());
+
+        address expectedAddress =
+            Create2Utils.computeCreate2Address(sender, salt, bytes32(bytecodeHash), constructorInputHash);
+
+        // deploy via create2
+        address actualAddress = address(new ConstantNumber{salt: salt}());
```

Comment:
```suggestion
        address actualAddress = address(new Greeter{salt: salt}());
```
You are using Greeter's hash as the expected address right?

Reply:
Good catch, bad copy-paste from the original test